app/cdap/components/common/DrawerWidget/index.tsx

```diff
@@ -126,7 +126,6 @@ export default function({
             <Label data-testid="widget-heading-text" component="span">
               {headingText}
             </Label>
-            {showHeaderSeparator && <img src="/cdap_assets/img/underline" />}
```

Comment:
Same as above. Should not remove as it is being used in **app/cdap/components/RecipeManagement/ApplyRecipeView.tsx** component

Reply:
Added it again

Reply:
As per v3 design we don't have underline img for Heading Title in entire application so removed showHeaderSeparator prop  as we don't require it anymore